### PR TITLE
pt_BR: Translate help messages

### PR DESCRIPTION
**Pull request by keysym**

### CODE REVIEW

**Comment by keysym** [comment]
Pinging @DatCrazyDelphox and @ojmarcelino so they can help reviewing, if they want to!

**Review by ojmarcelino — CHANGES_REQUESTED** [review]

File: po/pt_BR.po [hunk 1]
```diff
@@ -236,103 +247,464 @@ msgstr "asp retornou {}"
...
+"    --[no]useask           Automatically resolve conflicts using pacman's "
+"ask flag"
+msgstr ""
+"    --[no]useask           Resolve conflitos automaticamente solicitando "
+"confirmação do pacman"
+
+#: src/help.rs:65
+msgid "    --[no]newsonupgrade    Print new news during sysupgrade"
+msgstr ""
+"    --[no]newsonupgrade    Mostra novas notícias durante a atualização do "
```
[the remark this answers is not in the document]
Comment (keysym) [reply po/pt_BR.po]: Fixed! Great catch!

**Review by ojmarcelino — APPROVED** [review]

File: po/pt_BR.po [hunk 2]
```diff
@@ -236,103 +247,463 @@ msgstr "asp retornou {}"
...
+"    --clonedir  <diretório>      Diretório usado para baixar e executar "
+"PKGBUILDs"
+
+#: src/help.rs:36
+msgid "    --makepkg   <file>     makepkg command to use"
+msgstr "    --makepkg   <arquivo>     Comando makepkg a ser usado"
+
+#: src/help.rs:37
+msgid "    --mflags    <flags>    Pass arguments to makepkg"
+msgstr "    --mflags    <argumentos>    Passa argumentos para makepkg"
```
Comment (Morganamilo) [inline po/pt_BR.po:369]: This breaks alignment.

File: po/pt_BR.po [hunk 3]
```diff
@@ -236,103 +247,463 @@ msgstr "asp retornou {}"
...
+msgstr "    -a --aur               Assume que os alvos sejam do AUR"
+
+#: src/help.rs:33
+msgid "    --aururl    <url>      Set an alternative AUR URL"
+msgstr "    --aururl    <url>      Define uma URL alternativa do AUR"
+
+#: src/help.rs:34
+msgid "    --clonedir  <dir>      Directory used to download and run PKGBUILDs"
+msgstr ""
+"    --clonedir  <diretório>      Diretório usado para baixar e executar "
```
Comment (Morganamilo) [inline po/pt_BR.po:360]: This breaks alignment.

**Comment by keysym** [comment]
Fixed alignment:

```
Novas opções:
       --repo                   Assume que os alvos sejam dos repositórios
    -a --aur                    Assume que os alvos sejam do AUR
    --aururl    <url>           Define uma URL alternativa do AUR
    --clonedir  <diretório>     Diretório usado para baixar e executar PKGBUILDs

    --makepkg   <arquivo>       Comando makepkg a ser usado
    --mflags    <argumentos>    Passa argumentos para makepkg
    --pacman    <arquivo>       Comando pacman a ser usado
    --git       <arquivo>       Comando git a ser usado
    --gitflags  <argumentos>    Passa argumentos para git
    --sudo      <arquivo>       Comando sudo a ser usado
    --sudoflags <argumentos>    Passa argumentos para sudo
    --asp       <arquivo>       Comando asp a ser usado
    --bat       <arquivo>       Comando bat a ser usado
    --batflags  <argumentos>    Passa argumentos para bat
    --gpg       <arquivo>       Comando gpg a ser usado
    --gpgflags  <argumentos>    Passa argumentos para gpg
    --fm        <arquivo>       Gerenciador de arquivos a ser usado para revisão de PKGBUILD
    --fmflags   <argumentos>    Passa argumentos ao gerenciador de arquivos

```

**Review by ojmarcelino — APPROVED** [review]

**Review by ojmarcelino — APPROVED** [review]

**Pull request merged**